File: backend/Projetos/KeySystems.ERP.ContasAPagar/src/KeySystems.ERP.ContasAPagar.Core/DomainModel/ContasAPagar/TipoPgto.cs
namespace KeySystems.ERP.ContasAPagar.Core.DomainModel.ContasAPagar
{
    public class TipoPgto : IEntity<int>
    {
        private TipoPgto(){}
        public TipoPgto(string descricao)
        {
            this.Descricao = descricao;
        }

        public int Id { get; private set; }
        public string Descricao { get; private set; }
    }
}

File: frontend/src/app/modules/contas-pagar/titulo/shared/models/tipoPgto.ts


export class TipoPgto{
    
    constructor(descricao : string) {
        this.descricao = descricao;
    }
    
    private _id : number;
    public get id() : number {
        return this._id;
    }
    public set id(v : number) {
        this._id = v;
    }
    
    private _descricao : string;
    public get descricao() : string {
        return this._descricao;
    }
    public set descricao(v : string) {
        this._descricao = v;
    }
}